resources/Lessons/sources/introduction.md

```diff
@@ -0,0 +1,9 @@
+Hello 👋 and welcome to GLPI! This is the beginning of your adventure into the world of Free and OpenSource ITSM software designed and written by Teclib'. Throughout this wizard, I'll guide you step by step so that you get off to a good start and don't feel lost in this ocean of settings :).
```

Comment:
```suggestion
Hello 👋 and welcome to GLPI! This is the beginning of your adventure into the world of Free and Open Source ITSM software designed and written by Teclib'. Throughout this wizard, you will be guided step by step through some of the basic GLPI functionality.
```
Lesson text should not refer to itself as a person. I think the tone, besides the first (and maybe second) sentences, needs to be more formal too.